addons/spreadsheet_dashboard/security/security.xml

```diff
@@ -6,15 +6,15 @@
         <field name="domain_force">[('group_ids', 'in', user.groups_id.ids)]</field>
     </record>
 
-    <record model="ir.module.category" id="spreadsheet_dashboard.dashboard_management">
+    <record model="ir.module.category" id="base.module_category_productivity_dashboard">
```

Comment:
why `base...` ?

Reply:
can you also move the id before the model ?

Reply:
without the base prefix, the sequence applied to the ir.module.category is ignored. and here we need that sequence so that's why i used it.

Reply:
@adsh-odoo why ? the prefix is not correct...

Reply:
Hello @std-odoo 

In the module category record, we utilize the "base" prefix to ensure that the sequence specified within this record is honored. Omitting the "base" prefix would result in the sequence not being respected. This convention is consistently applied throughout the entire Odoo code base.

I have seen this thing in every module, That's why I have done this because I was also having the issue of the sequencing and it was resolved by this base prefix 

IMO -: we have to add the fields from multiple modules and their sequence should that be why we are using the base prefix but not 100% sure 
Correct me if I am wrong

Reply:
ah indeed, look like that's the way to do... thanks for the explanation